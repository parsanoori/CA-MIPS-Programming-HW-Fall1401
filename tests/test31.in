3
791905580
335789568
369397763