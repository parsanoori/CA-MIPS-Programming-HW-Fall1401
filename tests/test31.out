335789568
369397763
791905580